libkbfs/rekey_fsm.go

```diff
@@ -454,6 +460,19 @@ func NewRekeyFSM(fbo *folderBranchOps) RekeyFSM {
 
 		listeners: make(map[rekeyEventType][]rekeyFSMListener),
 	}
+	switch fbo.config.Mode() {
+	case InitDefault:
+		// In normal desktop app, we limit to 16 routines.
+		fsm.limiter = NewOngoingWorkLimiter(16)
```

Comment:
Isn't there a `rekeyFSM` per-`folderBranchOps`?  We need to limit rekeys across all the different folder branch ops, right?  I think a global limiter would need to be passed in to make that happen.